getting-started/deploy-to-dabl/6_start_web_server.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+cd /root/create-daml-app/target
+/usr/bin/python2 -m SimpleHTTPServer 8080 &
+sleep 1000
```

Comment:
Very ingenious way to offer download links. Why the `sleep 1000` here?

Reply:
if I don't sleep here the script gets terminated and the SimpleHTTPServer dies unfortunately..